README.md

```diff
@@ -153,9 +153,9 @@ There are several ways to install the Apache Release version of APISIX:
    - Download the latest source code release package:
 
      ```shell
-     $ mkdir apisix-2.4
-     $ wget https://downloads.apache.org/apisix/2.4/apache-apisix-2.4-src.tgz
-     $ tar zxvf apache-apisix-2.4-src.tgz -C apisix-2.4
+     $ mkdir apisix-2.5
```

Comment:
could we use an env here? or we have to change those versions one by one 🤔

Reply:
It is fine. I do the update with `sed`.